blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/AbstractBitbucketScmContentProvider.java

```diff
@@ -133,13 +134,15 @@ public GitContent getContent() {
     }
 
     @SuppressWarnings("unchecked")
-    @Override
-    public boolean support(@Nonnull Item item) {
+    @CheckForNull
```

Comment:
What's `@CheckForNull` actually mean? I googled it and came away more confused than when I started, which makes me sad. Is it just marking the return value as nullable?